src/toil/jobStores/abstractJobStore.py

```diff
@@ -866,3 +884,28 @@ def _defaultTryCount(self):
     @classmethod
     def _validateSharedFileName(cls, sharedFileName):
         return bool(cls.sharedFileNameRegex.match(sharedFileName))
+
+
+class SupportJobStore(AbstractJobStore):
+    @classmethod
+    def _supportsUrl(cls, url, method=methods.both):
+        def importOnly():
+            return url.scheme.lower() in ('http', 'https')
+
+        def exportOnly():
+            return False
+
+        def both():
+            return False
+
+        return cls._supportsUrlBody(method, importOnly, exportOnly, both)
```

Comment:
Much to complicated. _supportsUrlBody needs to go, sorry. Maybe Enum wasn't the right decision.

Do we really ever need to know if a URL is supported for both import and export? We are either importing or exporting, so when do we care if a batch system supports both modes?